src/main/resources/theme/fc-theme/login/messages/messages_en.properties

```diff
@@ -1,2 +1 @@
 fcInfos=learn more about France Connect
-

```

Comment:
emptyLine at the end of file is CRLF standard I think

Reply:
Done